testutil-server/src/test/java/io/spine/testing/client/grpc/TestClientTest.java

```diff
@@ -44,6 +45,7 @@
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
+@MuteLogging
```

Comment:
Not sure again. Is this due to a shutdown? Let's comment on muting in this case.